aiida/tools/importexport/dbexport/__init__.py

```diff
@@ -81,14 +84,92 @@ def export_zip(what, outfile='testzip', overwrite=False, silent=False, use_compr
         exporting.
     :raises `~aiida.common.exceptions.LicensingException`: if any node is licensed under forbidden license.
     """
-    if not overwrite and os.path.exists(outfile):
-        raise exceptions.ArchiveExportError("the output file '{}' already exists".format(outfile))
+    if file_format is None:
```

Comment:
Why not put `zip` as the default in the function definition?

Reply:
I could, I guess. I just have it in my head that setting defaults like that, instead of `None` can cause a usage issue - but that may be in different scenarios - like when setting a class attribute, and not in this case?